src/komponenter/kalkulator/utregninger.ts

```diff
@@ -221,14 +187,50 @@ export const datoIntervallErDefinert = (datoIntervall: DatoIntervall) => {
     );
 };
 
-export const konstruerStatiskTidslinje = (
+export const finnSluttDatoPåTidslinje = (
```

Comment:
Er det ikke dette `finnGrenserFor18MNDPeriode` skal gjøre? 🤔

Reply:
nei, dette skal finne grensa på hvor langt tidslinja strekker seg.  prøvde å refaktorere den til finnØvreGrenseForSluttdatoPåTidslinje. Er det bedre?

Reply:
Hva gjør `finnGrenserFor18MNDPeriode` da? 🤔

Reply:
den setter min-grense for 18 mnds-perioden ut ifra dagens dato. den bør renames.